### DISCUSSION TITLE
Do you have any plans for the next release?

### DISCUSSION THREAD

**Discussion by thaibm92** [arichorn/YouTubePlus#7] [General]
I tried updating for youtube v18.13.2 but got an error that can't save google account login information, exiting the application and re-entering it is lost (automatically logged out)
Waiting for your new release.

**Comment by arichornlover** [comment]
I didn’t see this discussion, sorry for not releasing fast enough. that’s what I get for focusing on the CercubePlus Tweak 🤷
